src/world/area_kmr/kmr_12/events.c

```diff
@@ -1,37 +1,33 @@
 #include "kmr_12.h"
 
-static Script make_entities;
-static Script read_west_sign;
-static NpcGroupList npc_groups;
+Script M(ExitWest) = EXIT_WALK_SCRIPT(60, 0, "kmr_07", 1);
+Script M(ExitEast) = EXIT_WALK_SCRIPT(60, 1, "kmr_11", 0);
 
-static Script exit_west = EXIT_WALK_SCRIPT(60, 0, "kmr_07", 1);
-static Script exit_east = EXIT_WALK_SCRIPT(60, 1, "kmr_11", 0);
-
-static Script bind_exits = {
-    SI_BIND(exit_west, TriggerFlag_FLOOR_ABOVE, 0 /* deili1 */, NULL),
-    SI_BIND(exit_east, TriggerFlag_FLOOR_ABOVE, 3 /* deili2 */, NULL),
+Script M(BindExits) = {
+    SI_BIND(M(ExitWest), TriggerFlag_FLOOR_ABOVE, 0 /* deili1 */, NULL),
+    SI_BIND(M(ExitEast), TriggerFlag_FLOOR_ABOVE, 3 /* deili2 */, NULL),
     SI_RETURN(),
     SI_END(),
 };
 
-Script kmr_12_main = {
+Script M(Main) = {
     SI_SET(SI_SAVE_VAR(425), 31),
     SI_CALL(SetSpriteShading, -1),
     SI_CALL(SetCamPerspective, 0, 3, 25, 16, 4096),
     SI_CALL(SetCamBGColor, 0, 0, 0, 0),
     SI_CALL(SetCamEnabled, 0, 1),
-    SI_CALL(MakeNpcs, 0, npc_groups),
-    SI_EXEC_WAIT(make_entities),
-    SI_EXEC(kmr_12_play_music),
-    SI_SET(SI_VAR(0), bind_exits),
+    SI_CALL(MakeNpcs, 0, M(npcGroupList)),
```

Comment:
Did we decide this should be `Npc` even at the beginning, or am I forgetting? So many "npc" naming conversations :p

Reply:
IMO PascalCase should be reserved from script API functions, scripts, and typedefs

Reply:
oh right, this is a variable. carry on